src/Data/Map/Internal.purs

```diff
@@ -158,6 +155,52 @@ instance traversableWithIndexMap :: TraversableWithIndex k (Map k) where
           <*> f k2 v2
           <*> traverseWithIndex f right
 
+-- | `SemigroupMap k v` provides a `Semigroup` instance for `Map` whose definition
+-- | depends on the underlying `value` type's `Semigroup` instance.
```

Comment:
What about

```
-- | `SemigroupMap k v` provides a `Semigroup` instance for `Map k v` whose definition
-- | depends on the `Semigroup` instance for the values.
```

? At least I think we should mention the type parameters of Map if we do for SemigroupMap and we should not quote “value“ (or we should rename `k` and `v` to `key` and `value`).

Reply:
I renamed `k v` to `key value`

Reply:
I’d prefer to stick with `k v` personally, since we are already using that almost completely uniformly in this library.

Reply:
Fixed.